wasm/src/lsp/parse.rs

```diff
@@ -1,10 +1,19 @@
-use std::collections::HashMap;
+use std::borrow::Borrow;
+use std::collections::{HashMap, HashSet};
 use std::rc::Rc;
-use clvm_tools_rs::compiler::comptypes::{BodyForm, CompileErr, CompileForm};
-use clvm_tools_rs::compiler::sexp::SExp;
+
+use lsp_types::Position;
+
+use clvm_tools_rs::compiler::compiler::DefaultCompilerOpts;
```

Comment:
`cargo build` warned me 'unused import'.
It seems this line should be annotated with `#[cfg(test)]`

Reply:
Thanks.